src/components/QuestionPlay/index.tsx

```diff
@@ -1,135 +1,156 @@
-                className={classNames({
-                  [styles.isAttempted]: isAttempted,
-                  [styles.correct]: option.isCorrect && selectedOptionId === option.optionId,
-                  [styles.inCorrect]: !option.isCorrect && selectedOptionId === option.optionId,
-                  [styles.isPreview]: isPreview,
-                })}
-                disabled={isAttempted}
-              />
-            ))}
-      </div>
-      {/* show reveal answer button for question without options in non preview mode when playing */}
-      {isWithoutOptions && !isPreview && !isAnswerRevealed && (
-        <Button
-          type="button"
-          color="blue"
-          onClick={() => {
-            setIsAnswerRevealed(true);
-
-            // pause timer when the answer is revealed
-            if (pauseTimer) {
-              pauseTimer();
-            }
-          }}>
-          Reveal answer
-        </Button>
-      )}
-      <Divider />
-      {isAttempted && (!isPreview || isQuestionSaved) ? (
-        <Button size="large" onClick={() => onClose()} type="button" color="blue">
-          Close
-        </Button>
-      ) : isWithoutOptions && !isPreview ? (
-        // Show correct & incorrect button in non preview mode when playing
-        // When answer is revealed by quiz host show correct or incorrect button, clicking on which will add points accordingly
-        isAnswerRevealed && (
-          <div className="flex">
-            <Button type="button" size="large" className="fullWidth" color="red" onClick={() => submitResponse(null)}>
-              Incorrect
-            </Button>
+              </Box>
+              {selectedOptionId === '' && (
+                <Group>
+                  <Button radius="md" color="red" onClick={() => handleSelectedChoice('')}>
+                    Incorrect
+                  </Button>
+                  <Button radius="md" color="green" onClick={() => handleSelectedChoice(options[0].optionId)}>
+                    Correct
+                  </Button>
+                </Group>
+              )}
+            </>
+          ) : (
             <Button
-              type="button"
-              size="large"
-              className="ml-lg fullWidth"
               color="green"
-              onClick={() => submitResponse(options[0].optionId)}>
-              Correct
+              radius="md"
+              onClick={() => {
+                setIsAnswerRevealed(true);
+
+                // pause timer when the answer is revealed
+                if (pauseTimer) {
+                  pauseTimer();
+                }
+              }}>
+              Reveal answer
             </Button>
-          </div>
-        )
-      ) : (
-        // for preview mode show save button & while playing show submit button for mcq
-        <Button type="submit" size="large" color="green">
-          {isPreview ? 'Save' : 'Submit'}
-        </Button>
-      )}
-    </form>
+          )
+        ) : (
```

Comment:
To make this component more readable can we split this into 2 components: `WIthOptions` & `WithoutOptions`?